VideoLAN.VLC version 3.0.18

borque opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90601)

borque committed:
VideoLAN.VLC version 3.0.18

mdanish-kh commented on manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:
```diff
@@ -0,0 +1,96 @@
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.msi
```
```suggestion
  InstallerUrl: https://download.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.msi
```
Please use the direct URL to the installer.

mdanish-kh commented on manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:
```diff
@@ -0,0 +1,96 @@
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.msi
```
```suggestion
  InstallerUrl: https://download.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.msi
```
Please use the direct URL to the installer.

mdanish-kh commented on manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:
```diff
@@ -0,0 +1,96 @@
+    InstallerType: msi
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.exe
```
```suggestion
  InstallerUrl: https://download.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.exe
```
Please use the direct URL to the installer.

mdanish-kh commented on manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:
```diff
@@ -0,0 +1,96 @@
+    InstallerType: nullsoft
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.exe
```
```suggestion
  InstallerUrl: https://download.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.exe
```
Please use the direct URL to the installer.

mdanish-kh commented on manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:
```diff
@@ -0,0 +1,96 @@
+  InstallerSha256: A618D66B2F753343402200AD17D92EB6D4AB10DB0DD1A098402E2968AA3114C4
+  ProductCode: '{B022B1C5-D067-42CB-98E2-D965E4D74CFE}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player 3.0.18 (64-bit)
```
```suggestion
  - DisplayName: VLC media player
```
Must match Apps&Features/Control Panel.

borque replied:
Ahh, allright, I "reused" this from the previous manifest and simply adjusted the version number...

mdanish-kh commented on manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:
```diff
@@ -0,0 +1,96 @@
+  InstallerSha256: 72EB05D21A4159450DBB39F2A09C5ED7C8370C9B4CAD75CE36A1A71B9F8A7D11
+  ProductCode: '{6799A970-32B3-47B4-89AA-2DF0806FB862}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player 3.0.18 (32-bit)
```
```suggestion
  - DisplayName: VLC media player
```
Must match Apps&Features/Control Panel.

mdanish-kh commented on manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:
```diff
@@ -0,0 +1,96 @@
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player 3.0.18 (32-bit)
+    Publisher: VideoLAN
+    DisplayVersion: 3.0.18
```
```suggestion
    DisplayVersion: 3.0.18.0
```
Must match Apps&Features/Control Panel.

mdanish-kh commented on manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:
```diff
@@ -0,0 +1,96 @@
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player
+    Publisher: VideoLAN
+    DisplayVersion: 3.0.18
```
```suggestion
```
DisplayVersion should not be used when it is equal to PackageVersion.

mdanish-kh commented on manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:
```diff
@@ -0,0 +1,96 @@
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player
+    Publisher: VideoLAN
+    DisplayVersion: 3.0.18
```
```suggestion
```
DisplayVersion should not be used when it is equal to PackageVersion.

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

When I created the manifest, I used https://download..., but I got a 404. Well, tried it again and now it's working... Here we go...

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

When I created the manifest, I used https://download..., but I got a 404. Well, tried it again and now it's working... Here we go...

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

When I created the manifest, I used https://download..., but I got a 404. Well, tried it again and now it's working... Here we go...

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

When I created the manifest, I used https://download..., but I got a 404. Well, tried it again and now it's working... Here we go...

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

Ahh, allright, I "reused" this from the previous manifest and simply adjusted the version number...

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>

borque committed:
Update manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

Ahh, allright, I "reused" this from the previous manifest and simply adjusted the version number...

Co-authored-by: Muhammad Danish <mdanishkhdev@gmail.com>
